_layouts/packages.html

```diff
@@ -4,16 +4,24 @@
 
 {% capture manual %}{{ site.baseurl }}/Manuals/pkg/{{ page.dirname }}{% endcapture %}
 
-<h2>GAP package {{ page.name }}</h2>
+<h1>GAP package {{ page.name }}</h1>
 
 <h2>{{ page.description }}</h2>
 
-<p class="homelink">[<a href="{{ page.www }}">WWW homepage</a>]</p>
+<h4>Short Description</h4>
+<p>{{ page.abstract }}</p>
+
+<p class="homelink">[<a href="{{ page.www }}">WWW homepage</a>]  &nbsp; &nbsp; &nbsp;
+    {% if page.source-repo %} [<a href="{{ page.source-repo.url }}"> Source code repository</a>] {% endif %}
+    &nbsp; &nbsp; &nbsp;
+    {% if page.issue-url %} [<a href="{{ page.issue-url }}">Issue tracker</a>] {% endif %}
 
+</p>
+<p>
 {% if page.authors.size > 1 %}
-    <h4>Authors</h4>
+    <b>Authors</b> &nbsp;
```

Comment:
Make sure to remove the `&nbsp;` here when you change to a description list

Reply:
Yep caught that as it looked hideous.